0.data_analysis/plate_5_characteristics/nbconverted/plate_5_sc_feature_selected_camerons_agg_well_corr.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+import sys
+
+import pandas as pd
+
+# ## Find the root of the git repo on the host system
+
+# In[2]:
+
+
+# Get the current working directory
+cwd = pathlib.Path.cwd()
+
+if (cwd / ".git").is_dir():
+    root_dir = cwd
+
+else:
+    root_dir = None
+    for parent in cwd.parents:
+        if (parent / ".git").is_dir():
+            root_dir = parent
+            break
+
+# Check if a Git root directory was found
+if root_dir is None:
+    raise FileNotFoundError("No Git root directory found.")
+
+
+# ## Define paths
+
+# ### Input paths
+
+# In[3]:
+
+
+# Path to correlation class
+sys.path.append(
+    f"{root_dir}/0.data_analysis/plate_5_characteristics/utils"
+)
+
+platedf_path = pathlib.Path(root_dir / "nf1_painting_repo/3.processing_features/data/single_cell_profiles/Plate_5_sc_feature_selected.parquet").resolve(strict=True)
+platedf = pd.read_parquet(platedf_path)
+
+# Class for calculating correlations
+from CorrelatePlate import CorrelatePlate
```

Comment:
consider importing this class at the top of the script and commenting it as  class

Reply:
I though about that, I use the `root_dir` to import the `CorrelatePlate` class, so it would have to go below the code where the `root_dir` is defined